cli/js/web/console.ts

```diff
@@ -14,17 +14,23 @@ import {
   bold,
 } from "../colors.ts";
 
+// TODO(nayeemrmn): Improve options handling in this module. Less positional
```

Comment:
@nayeemrmn I think it's better to clean this up in this PR, else it's going to be a technical debt. Just wrap those options into a single object.

Reply:
Yeah no worries, I meant to. It'll be at least a week before this can land due to minor API changes.

Reply:
Nice 👍